test/GogeDao.t.sol

```diff
@@ -319,19 +407,682 @@ contract DaoTest is Utility, Test {
+        assertEq(gogeDao.totalVotes(1), joe_votes + 1 ether + gogeDao.minAuthorBal());
+        assertEq(gogeDao.passed(1), true);
+        assertEq(gogeDao.pollEndTime(1), block.timestamp);
+
+        // Verify poll has not been executed this Joe is not blacklisted, yet.
+        assertEq(gogeToken.isBlacklisted(address(joe)), true);
+    }
+
+    /// @notice Verify when payTeam is called the addresses held in teamMembers are paid out
+    function test_gogeDao_payTeam() public {
+        emit log_named_uint("BNB Balance", address(this).balance); // 79228162214.264337593543950335
+
+        // update governanceTokenAddr to address(this)
+        test_gogeDao_updateGovernanceToken();
+
+        // add team members
+        gogeDao.setTeamMember(address(jon), true);
+        gogeDao.setTeamMember(address(tim), true);
+
+        // update team balance on dao
+        payable(address(gogeDao)).transfer(1 ether);
+        gogeDao.updateTeamBalance(1 ether);
+
+        // Pre-state check
+        assertEq(address(gogeDao).balance, 1 ether);
+        assertEq(address(gogeDao).balance, gogeDao.teamBalance());
+        assertEq(address(jon).balance, 0 ether);
+        assertEq(address(tim).balance, 0 ether);
+
+        // pay team
+        gogeDao.payTeam();
+
+        // Post-state check
+        assertEq(address(gogeDao).balance, 0 ether);
+        assertEq(address(gogeDao).balance, gogeDao.teamBalance());
+        assertEq(address(jon).balance, 0.5 ether);
+        assertEq(address(tim).balance, 0.5 ether);
+
+    }
+
+    /// @notice Verify correct logic when payTeam is called with a range of amounts using fuzzing.
+    function test_gogeDao_payTeam_fuzzing(uint256 _amount) public {
+        emit log_named_uint("BNB Balance", address(this).balance);
+
+        _amount = bound(_amount, 0, address(this).balance);
+
+        // update governanceTokenAddr to address(this)
+        test_gogeDao_updateGovernanceToken();
+
+        // add team members
+        gogeDao.setTeamMember(address(jon), true);
+        gogeDao.setTeamMember(address(tim), true);
+        gogeDao.setTeamMember(address(joe), true);
+
+        // update team balance on dao
+        payable(address(gogeDao)).transfer(_amount);
+        gogeDao.updateTeamBalance(_amount);
+
+        // Pre-state check
+        assertEq(address(gogeDao).balance, _amount);
+        assertEq(address(gogeDao).balance, gogeDao.teamBalance());
+        assertEq(address(jon).balance, 0 ether);
+        assertEq(address(tim).balance, 0 ether);
+        assertEq(address(joe).balance, 0 ether);
+
+        // pay team
+        gogeDao.payTeam();
+
+        // Post-state check
+        assertEq(address(gogeDao).balance, 0 ether);
+        assertEq(address(gogeDao).balance, gogeDao.teamBalance());
+        assertEq(address(jon).balance, _amount / 3);
+        assertEq(address(tim).balance, _amount / 3);
+        withinDiff(address(joe).balance, _amount / 3, 2);
```

Comment:
Is there a reason `withinDiff` is used here instead of `assertEq`?

Reply:
Will show you once we meet up. Same thing as comment right above